bitnami/kafka/values.yaml

```diff
@@ -942,6 +950,14 @@ broker:
   ## @param broker.lifecycleHooks lifecycleHooks for the Kafka container to automate configuration before or after startup
   ##
   lifecycleHooks: {}
+  ## Kafka init container resource requests and limits
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param initcontroller.resources.limits The resources limits for the container
+  ## @param initcontroller.resources.requests The requested resources for the container
+  ##
```

Comment:
```suggestion
  ## Kafka init container resource requests and limits
  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
  ## @param broker.initContainerResources.limits The resources limits for the container
  ## @param broker.initContainerResources.requests The requested resources for the container
  ##
```

Reply:
Hi @dgomezleon 
Since this is the first time I submitted a PR to you, I did not pay attention to many regulations. I'm sorry. I have modified